### DISCUSSION TITLE
Thanks a lot for this Tool

### DISCUSSION THREAD

**Discussion by ClaudioFoscan** [maehw/WebPBrick#9] [General]
Hi maehw

jsut want to thank your for this tool, took out my old rcx and had to set up some winXP on a VM to run roboinvention System and the lot ... now its possible to code with scratch and on modern pcs... thanks for your effort ... will post some projects from my son soon....

Danke!

**Comment by maehw** [comment]
Hi @ClaudioFoscan ,

thanks for your positive feedback! It's always nice to hear that someone might actually find this useful. 

Have you already tested/used WebPBrick? Because IIRC the currently committed version didn't work locally for me and WebPBrick.com is still in the last release version (v0.2). So if you'd like to use the most recent version, some big still needs to be found and fixed. 

Sure, I'd love to see some ideas/projects shared with me and the world!

Cheers & bitte, gerne!
